checks.d/docker.py

```diff
@@ -16,6 +17,8 @@
 
 EVENT_TYPE = SOURCE_TYPE_NAME = 'docker'
 
+ECS_PORT = '51678/tcp'
```

Comment:
Do we need to hardcode this ? There is no way to get it from the docker api directly ?